client/src/extension.ts

```diff
@@ -108,48 +106,39 @@ export function activate(context: ExtensionContext) {
 		folder = getOuterMostWorkspaceFolder(folder);
 		
 		if (!clients.has(folder.uri.toString())) {
-			Promise.all(fileSearchExtensions.map(type => Workspace.findFiles(`**/*${type}`, '')))
+			Workspace.findFiles(`{${(peekToInclude || []).join(',')}}`, `{${(peekToExclude || []).join(',')}}`,)
```

Comment:
one of these is a list of languages, the other is a list of filetypes, so I don't think this would work 🤔

Reply:
I have it converting to globs a bit above but I could make a language to extensions map if needed